config.xml

```diff
@@ -1,70 +1,73 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
Hey,
this file is modified by the ionic cli, I didn't noticed this change.
I just used --save and it will save the plugin in config.xml (with this): 
`ionic plugin add https://github.com/apache/cordova-plugin-whitelist.git --save`
 
you can change it if you want but ofcourse when someone use this command it will still change